lib/augmentConfig.cjs

```diff
@@ -2,7 +2,7 @@
 // please instead edit the ESM counterpart and rebuild with Rollup (npm run build).
 'use strict';
 
-const node_path = require('node:path');
+const path = require('node:path');
```

Comment:
minor

Even if I agree with the rename, I think you should reduce the diff if possible.
i.e. minor refactors should be avoided if you don't change anything in a file apart from the rename